<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternalPlotDialog</class>
 <widget class="QDialog" name="ExternalPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>External Plot Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFileSelect">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot Capture</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPlotCapture">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Grid size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>x-axis</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxXMin"/>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBoxXMax"/>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="spinBoxXGridSize"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>y-axis (left)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBoxYLeftMin"/>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBoxYLeftMax"/>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="spinBoxYLeftGridSize"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>y-axis (right)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBoxYRightMin"/>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="spinBoxYRightMax"/>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="spinBoxYRightGridSize"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>369</width>
     <height>174</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="5">
     <widget class="QCheckBox" name="checkBoxCurve1Cross">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QCheckBox" name="checkBoxCurve3Line">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2" rowspan="2">
     <widget class="QComboBox" name="comboBoxCurve2YSide">
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>y-axis (R,L)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Curve 4:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Cross</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Legend</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QComboBox" name="comboBoxCurve4YSide">
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Line</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineEditCurve1Legend"/>
    </item>
    <item row="3" column="2" rowspan="2">
     <widget class="QComboBox" name="comboBoxCurve1YSide">
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBoxXCol"/>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="checkBoxCurve1Line">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Curve 2:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Curve 1:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="lineEditCurve2Legend"/>
    </item>
    <item row="7" column="5">
     <widget class="QCheckBox" name="checkBoxCurve3Cross">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBoxCurve1Col"/>
    </item>
    <item row="4" column="1" rowspan="2">
     <widget class="QSpinBox" name="spinBoxCurve2Col"/>
    </item>
    <item row="9" column="3">
     <widget class="QLineEdit" name="lineEditCurve4Legend"/>
    </item>
    <item row="5" column="4">
     <widget class="QCheckBox" name="checkBoxCurve2Line">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QCheckBox" name="checkBoxCurve2Cross">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1" rowspan="2">
     <widget class="QSpinBox" name="spinBoxCurve3Col"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Curve 3:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" rowspan="2">
     <widget class="QComboBox" name="comboBoxCurve3YSide">
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="lineEditCurve3Legend"/>
    </item>
    <item row="9" column="4">
     <widget class="QCheckBox" name="checkBoxCurve4Line">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QCheckBox" name="checkBoxCurve4Cross">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="1" rowspan="2">
     <widget class="QSpinBox" name="spinBoxCurve4Col"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Column (1-20)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExternalPlotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExternalPlotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFileSelect</sender>
   <signal>clicked()</signal>
   <receiver>ExternalPlotDialog</receiver>
   <slot>fileSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileSelect()</slot>
 </slots>
</ui>
